### ISSUE TITLE
Option to disable temporary headers in code generator

### ISSUE THREAD

**Issue by saswatds** [a85/POSTMan-Chrome-Extension#6680]
**Is your feature request related to a problem? Please describe.**
The temporary headers have been added to UI to represent the actual behavior of postman but as a side-effect the headers are now also a part of code-generator. A large group of users may not want to have these headers as part of the generated code and only would want to see what they have explicitly added. 

**Describe the solution you'd like**
An option in the code-generator or in global settings to disable adding of temporary headers in code-generator.

**Additional context**
[Filed by postman]

**Comment by shreys7** [comment]
@saswatds In Postman 7.2.1, we've introduced bug-fixes, support for two new languages (JS-Fetch and Powershell), and the ability to configure certain aspects of the generated snippet with the 'Experimental Codegen Mode' flag enabled in Postman's global settings. You can switch off the include temporary headers option in configurations of the snippet generated.

The options can be seen with the 'Experimental Codegen mode' flag set.
![image](https://user-images.githubusercontent.com/20665864/59862143-dd190b80-939f-11e9-85a8-a5875abbc6d3.png)

![image](https://user-images.githubusercontent.com/20665864/59862093-be1a7980-939f-11e9-8622-91b07f119743.png)

**Comment by berend** [comment]
I used the above described settings, those headers are gone now. But now curl code is generated with `"` as string delimiter for the `-d`/`--data` option. Since I guess most of the people here send json, which also requires `"` as string delimiter, all `"` from the json are now escaped via `\"`. The code still works, but it looks ugly.

Without that experimental flag, curl code is generated with `'` as string delimiter for curls `-d`/`--data` option. json `"` quote do not need escaping, looks way better.

I looked briefy into the code, it must be around [this block](https://github.com/postmanlabs/postman-code-generators/blob/8bb9ba398a74954cd65cda7dd5e6f0963cb55470/codegens/curl/lib/index.js#L47-L86)
